### ISSUE TITLE
Can't check unimported files in other folders if "src" folder exists

### ISSUE THREAD

**Issue by deviant310** [smeijer/unimported#171]
Hello!

I have a react project.
I want to check unimported files in another sibling folder, not in `src` folder (only in `build` folder in my case).
I can do it only after renaming `src` folder. If I won't do this, module will search unimported files only in `src` folder.

I don't mind this behavior by default, but I would like that it be configurable.
Thanks.

**Comment by ritingliudd01** [comment]
This `src` will become configurable when this PR https://github.com/smeijer/unimported/pull/168 is approved :)

**Comment by deviant310** [comment]
@smeijer, waiting for your approve, thanks:)

**Comment by ritingliudd01** [comment]
@deviant310 
That PR is merged. Please try the latest release.

**Comment by smeijer** [comment]
fixed by #168

**Comment by smeijer** [comment]
fixed by #168
